build.gradle

```diff
@@ -92,7 +90,7 @@ subprojects {
 	}
 
 	dependencyManagement {
-		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}" //mavenBom "org.springframework.cloud:spring-cloud-starter-parent:${springCloudStarterParentTrain}"
+		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springCloudStarterParentTrain}"
```

Comment:
Yes, check it out!
https://spring.io/blog/2015/11/25/migrating-spring-cloud-apps-from-spring-boot-1-2-to-1-3

On Mon, Nov 30, 2015 at 6:10 AM, David Ferreira Pinto <
notifications@github.com> wrote:

> In build.gradle
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43#discussion_r46135839>
> :
>
> > @@ -92,7 +90,7 @@ subprojects {
> >  	}
> >
> >  	dependencyManagement {
> > -		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}" //mavenBom "org.springframework.cloud:spring-cloud-starter-parent:${springCloudStarterParentTrain}"
> > +		imports { mavenBom "org.springframework.boot:spring-boot-starter-parent:${springCloudStarterParentTrain}"
>
> is there a Brixton.M3 version of the spring-boot-starter-parent?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43/files#r46135839>
> .
>



-- 
Corby Page
Senior Field Engineer, Pivotal
713.584.3168